2.Arrays/arrays.js

```diff
@@ -0,0 +1,102 @@
+function isArray1(arr) {
+	return arr instanceof Array;
+}
+
+console.log(isArray1([3,null,undefined,4]));
+
+function isArray2(arr) {
```

Comment:
Which of these? I`ve found both in "JavaScript. The Definitive Guide" by David Flanagan, chapter 7.10.